generator/templates/Authorisation/src/templates/Authorisation.vue

```diff
@@ -1,13 +1,9 @@
 <template>
-    <div id="app">
-        <v-app class="primary" id="inspire">
-            <v-content>
-                <v-container fill-height fluid justify-center class="primary">
-                    <router-view/>
-                </v-container>
-            </v-content>
-        </v-app>
-    </div>
+    <v-main>
+        <v-container fill-height fluid justify-center class="primary">
```

Comment:
```suggestion
        <v-container fill-height fluid justify="center" class="primary">
```
Ik neem aan dat dit vuetify v2 is

Reply:
Ik denk zelfs dat fluid enzo niet meer bestaan

Reply:
Fluid has always existed for containers, and will keep existing. However, justify is not a thing for containers at all, the legacy documentation still contains this, but has gotten completely removed as from 2.0 (i think). Same goes for fill-height.